EthOn_glossary.md

```diff
@@ -273,6 +276,9 @@ Relates a Post Transaction State to the following Post Transaction State.
 Relates a State to the following State. In EthOn the state transition system has no branches.
 ## O
 ## P
+### Prefunded Account
+[`ethon:PrefundedAccount`](http://ethon.consensys.net/PrefundedAccount)   
+A Prefunded Account is an Account whose initial balance is defined in the protocol of the Blockchain. In the initial Ethereum Blockchain those that participated in the crowd funding received Prefunded Accounts. However, Prefunded Accounts could be added with any hard fork.
```

Comment:
The specification of the prefund balances and any other special account properties are part of the protocol specification - not irregular, imho.
The best concept I came up with, so far is introduction of a `hasProtocolState` and a `Protocol State`. This state can be created after initial protocol definition or any protocol change (`ethon:conformsTo`, `ethon:ProtocolVariant`). This solution is elegant because it allows for reuse of the state properties of normal Accounts.